api/src/routing.js

```diff
@@ -373,6 +376,7 @@ app.get('/api/v1/forms/', (req, res) => {
 });
 app.get('/api/v1/forms', forms.list);
 app.get('/api/v1/forms/:form', forms.get);
+app.post('/api/v1/forms/validate', textParser, forms.validate);
```

Comment:
Just a thought! 
I know the other forms endpoints are not password protected (because of medic-collect), but I think this endpoint should require authentication. 
Are we allowing unauthenticated users to flood us with XMLs to be parsed and tested?